docs/DeveloperGuide.md

```diff
@@ -585,7 +585,11 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 * **Dorm manager**: User of Dormanager Pro that has to keep track of the residents in their dorm
 * **Profile**: Collection of information related to a resident that serves as a block of interrelated data in Dormanger Pro. Consists of name, contact number, room number, and emergency contact.
 * **Emergency contact**: Person to contact when the resident related to said contact gets into an emergency (injury, immigration related issues etc.). Consists of a name and contact number.
-* **Dorm room**: Rooms of the dorm where residents stay in. Has a room number and upper limit of
+* **Dorm room**: Rooms of the dorm where residents stay in. Corresponds to a floor and room number that specify its location.
```

Comment:
Would it be better if we used "floor and unit number" instead, considering that we use Room Number to specify the floor number as well?